### DISCUSSION TITLE
[Question]: Accessibility of old events in 0.14

### DISCUSSION THREAD

**Discussion by BBaoVanC** [blakeblackshear/frigate#11965] [Ask A Question] [answered]
### What is your question:

I see in the changelogs for the new 0.14 betas it says,

> Your existing events will not be visible in Frigate's new UI
> - An entirely new way of reviewing camera footage has been implemented in Frigate 0.14, and existing events could not be migrated

Does this mean the events from previous versions will be inaccessible in 0.14, or just that you'll have to go to a different tab of the UI? The changelog says that you should export starred events, but if there's an incompatibility, why not just keep the existing 0.13 UI in a different tab, so you can still see all the old events in addition to using the new UI for new events?

Also, why can these old events not be migrated? I thought that old events would have sufficient information stored. If it's a question of developer workload, I'd love to look into it and potentially make a pull request.

**Comment by hawkeye217** [comment, marked as answer]
The new format for reviewing footage is completely different from the old "events" in 0.13. There's no easy way to migrate them so that they would be useful or helpful in any way. So no, "events" or "clips" from any previous versions are not accessible anywhere in 0.14.

Any recordings from prior versions are still accessible from the UI via the History button after clicking on an individual camera's live view.

**Reply by NickM-27** [reply]
And to be clear this is not a data format but rather a media format. The media that is used in the 0.14 UI (previews, review thumbnails, etc) did not exist in previous versions, so it can not be migrated.

**Reply by BBaoVanC** [reply]
Why not just display the best you can in the new UI, even if it doesn't have every single feature that the new UI does? Or at the very least, keep a tab to at least view old clips in the old UI so I don't lose them completely?

**Reply by NickM-27** [reply]
It's not like a small feature wouldn't work, the entire UI would not be comparable. Also, the UI was rewritten from the ground up. It would take a lot of work, testing, and not to mention user explanation of why this extra screen exists all just so the existing events would be visible for a few days until the recordings are deleted anyway. Of the thousands of users that are using the beta, we have not gotten a single complaint about this.

**Reply by hawkeye217** [reply]
We've chosen not to do that as part of the architecture and design decisions for 0.14.

Again, to be clear, all of your footage remains viewable through the History tab.

**Reply by BBaoVanC** [reply]
> Of the thousands of users that are using the beta, we have not gotten a single complaint about this.

That is precisely why I wanted to ask this.

I must have misunderstood what you said earlier since I was imagining that only old recordings would be visible, not clips. Does 'recordings' include the clips? I apologize for not just updating and seeing for myself, but I was worried about things being deleted and/or becoming inaccessible and I don't really have the space currently to make a full backup copy (but I will be clearing up some space in a little bit). Do I just need to make a backup of the SQLite database to keep the clips (the metadata, like start and end time and all), and the video files won't be deleted?

I don't quite follow what you mean about user explanation as presumably that extra screen could only be visible to people that already had clips from the old format and new users would not see it.

But since the UI was rewritten and it would take a lot of effort to port the old UI, that's fine.

**Reply by NickM-27** [reply]
"clips" as a concept in frigate has not existed for some time. both continuous recordings and events are all just recordings with metadata that says when various things (motion, events, etc.) started and stopped. so anything that was recorded is still visible in the new UI

**Comment by mrplow** [comment]
One could always run a 2nd instance of frigate at version 13. Just use different ports and update the config to have all the cameras using a black mp4 as the path and turn if detection and recording. 

Then you can view your old events like before 🤷‍♂️
